crates/wasm/fs/public/file.rs

```diff
@@ -21,12 +24,67 @@ impl PublicFile {
         let cid = Cid::try_from(&cid.to_vec()[..])
             .map_err(|e| Error::new(&format!("Invalid CID: {e}")))?;
 
-        Ok(PublicFile(WnfsPublicFile::new(time, cid)))
+        Ok(PublicFile(Rc::new(WnfsPublicFile::new(time, cid))))
     }
 
     /// Gets a unique id for node.
     #[wasm_bindgen(js_name = "getId")]
     pub fn get_id(&self) -> String {
         self.0.get_id()
     }
+
+    /// Stores a file in provided block store.
+    pub fn store(&self, store: BlockStore) -> JsResult<Promise> {
+        let file = Rc::clone(&self.0);
+        let mut store = ForeignBlockStore(store);
+
+        Ok(future_to_promise(async move {
+            let cid = file
+                .store(&mut store)
+                .await
+                .map_err(|e| Error::new(&format!("Cannot add to store: {e}")))?;
+
+            let cid_u8array = Uint8Array::from(&cid.to_bytes()[..]);
+
+            Ok(value!(cid_u8array))
+        }))
+    }
+
+    /// Loads a file given its CID from the block store.
+    pub fn load(cid: Uint8Array, store: BlockStore) -> JsResult<Promise> {
+        let store = ForeignBlockStore(store);
+        let cid = Cid::read_bytes(cid.to_vec().as_slice())
+            .map_err(|e| Error::new(&format!("Cannot parse cid: {e}")))?;
```

Comment:
The `cid` param can have a `Vec<u8>` type.

```suggestion
    pub fn load(cid: Vec<u8>, store: BlockStore) -> JsResult<Promise> {
        let store = ForeignBlockStore(store);
        let cid = Cid::try_from(cid)
            .map_err(|e| Error::new(&format!("Cannot parse cid: {e}")))?;
```